components/kyma-environment-broker/internal/broker/instance_update_test.go

```diff
@@ -86,4 +123,26 @@ func TestUpdateEndpoint_Update(t *testing.T) {
 	assert.Equal(t, internal.ERSContext{
 		Active: ptr.Bool(false),
 	}, handler.ersContext)
+	// check if handler was called with Instance.active=true
```

Comment:
this comment is kinda misleading - we call update with `active: false` payload and so is handler called